README.md

```diff
@@ -45,6 +45,7 @@ For Ginkgo core library:
 
 The Ginkgo CUDA module has the following __additional__ requirements:
 
+*   _Compute Capability 5.3+_
```

Comment:
There is no compute capability 5.3
```suggestion
*   _Compute Capability 6.0+_
```

Reply:
This is probably going to be removed, since @yhmtsai is working on a fix for that case.